CONTRIBUTING.md

```diff
@@ -33,6 +33,17 @@ maximize the chances of your PR being merged.
   deprecations between 1.3.0 and 1.4.0 will be deleted soon AFTER 1.5.0 is tagged and released
   (at the beginning of the 1.6.0 release cycle). This results in a three to six month window for
   migrating from deprecated code paths to new code paths.
+* Unless the community and Envoy maintainer team agrees on an exception, during the
+  first release cycle after a feature has been deprecated, use of that feature
+  will cause a logged warning, and incrementing the
+  [runtime](https://www.envoyproxy.io/docs/envoy/latest/configuration/runtime#config-runtime)
```

Comment:
nit: This feels like maybe a server stat vs. a runtime stat?

Reply:
No I think we definitely need the stat. I understand why you put it here and sure this seems fine given the constraints.